src/main/java/org/example/Properties/Ticket/TicketCreateParams.java

```diff
@@ -0,0 +1,79 @@
+package org.example.Properties.Ticket;
+
+import java.util.Optional;
+import java.util.OptionalInt;
+
+public class TicketCreateParams {
+    Optional<String> type = Optional.empty();
+    OptionalInt useCount = OptionalInt.empty();
+    OptionalInt writeDataObjectCount = OptionalInt.empty();
+    OptionalInt writeByteCount = OptionalInt.empty();
+    OptionalInt secondsUntilExpiration = OptionalInt.empty();
+    Optional<String> users = Optional.empty();
+    Optional<String> groups = Optional.empty();
+    Optional<String> hosts = Optional.empty();
```

Comment:
Looks like you're missing the `private` access specifier?

Please double check other types like this.

Reply:
Oops I did forget it. It's been added to all places that was missing it.